lib/spack/spack/binary_distribution.py

```diff
@@ -526,7 +526,7 @@ def extract_tarball(spec, filename, allow_root=False, unsigned=False,
     # Delay creating spec.prefix until verification is complete
     # and any relocation has been done.
     else:
-        install_tree(workdir, spec.prefix, symlinks=True)
+        shutil.copytree(workdir, spec.prefix, symlinks=True)
```

Comment:
For some reason, using the new `install_tree` wrapper here caused `test_buildcache` to crash. I have no idea why.